inc/policyboolean.class.php

```diff
@@ -79,10 +79,19 @@ public function getMqttMessage($value, $itemtype, $itemId) {
       return $array;
    }
 
+   /**
+    * Shows the input values
+    * @return mixed the values
+    */
    public function showValueInput() {
       return Dropdown::showYesNo('value', '0', -1, array('display' => false));
    }
 
+   /**
+    * Shows the values searched
```

Comment:
same here: value 

also this is not a search, it simply converts a value into a human readable form